tests/test_github_repo_preflight.py

```
@@ -0,0 +1,43 @@
+    proxy.app.dependency_overrides[direct_chat._get_current_user] = _fake_user
+
+    # Ensure git binary appears present
+    import shutil
+    monkeypatch.setattr(shutil, "which", lambda name: "/usr/bin/git")
+
+    # Return a token
+    monkeypatch.setattr(direct_chat, "_get_github_token_for_user", lambda email: "ghp_FAKE")
+
+    # Simulate git ls-remote failing by patching subprocess.run
+    def fake_run(cmd, stdout, stderr, env, timeout):
+        class P: returncode=128; stderr=b"fatal: Authentication failed"; stdout=b""
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)